try { () } catch err:* { "Division by zero" }, 1 div 0
